packages/blade/src/components/SkipNav/index.ts

```diff
@@ -0,0 +1 @@
+export * from './SkipNav.web';
```

Comment:
put this export under `src/components/index.ts` too so that this will be available for consumers too

Reply:
Right.